plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowArchetype.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.eclipse.dataflow.core.project;
+
+import java.util.NavigableSet;
+
+import com.google.common.collect.ImmutableSortedSet;
+
+/**
+ * Enumeration of the Archetype templates available for project creation.
+ */
+public enum DataflowArchetype {
```

Comment:
Makes sense. Done. I initially wanted to call this simply Archetype, but that was taken.